nav2_controller/CMakeLists.txt

```diff
@@ -72,13 +99,20 @@ install(TARGETS ${executable_name}
 install(DIRECTORY include/
   DESTINATION include/
 )
+install(FILES plugins.xml
```

Comment:
This isn't necessary anymore, `pluginlib_export_plugin_description_file` covers

Reply:
Removed